src/main/java/com/exasol/adapter/request/parser/PushdownSqlParser.java

```diff
@@ -364,12 +364,22 @@ private DataType getDataType(final JsonObject dataType) {
         case "GEOMETRY":
             type = getGeometry(dataType);
             break;
+        case "HASHTYPE":
```

Comment:
Remove breaks and return type immediately. Allows immutability.

Reply:
done